libs/sdk/README_SDK_SERVICES.md

```diff
@@ -1024,6 +1024,63 @@ disconnect(): Promise<void>
 
 **Returns**: A promise that resolves when the disconnection from Web3 provider is complete.
 
+### getSigner
+
+Gets the signer (a wallet instance) for signing messages or transactions. It throws a warning through the logger, if no Web3 provider is available.
+
+```ts
+// ...
+const web3Connector = new Web3Connector(...);
```

Comment:
you don't need to instantiate the class. access it using the already instantiated sdk object instead. `sdk.services.web3`?